src/components/OpenChannelModal/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { useState } from 'react';
+import { Dialog, DialogTitle, DialogContent, TextField, DialogActions, InputAdornment } from '@mui/material';
+import { useAppDispatch, useAppSelector } from '../../store';
+import { actionsAsync } from '../../store/slices/node/actionsAsync';
+
+type OpenChannelModalProps = {
+  handleRefresh: () => void;
+};
+
+export const OpenChannelModal = ({ handleRefresh }: OpenChannelModalProps) => {
+  const dispatch = useAppDispatch();
+  const loginData = useAppSelector((selector) => selector.auth.loginData);
+  const [openChannelModal, set_openChannelModal] = useState(false);
+  const [peerId, set_peerId] = useState('');
+  const [amount, set_amount] = useState('');
+
+  const handleOpenChannelDialog = () => {
+    set_openChannelModal(true);
+  };
+
+  const handleCloseChannelDialog = () => {
+    set_openChannelModal(false);
+  };
+
+  const handleOpenChannel = (amount: string, peerId: string) => {
+    // Close Modal after starting to open channel
+    handleCloseChannelDialog();
+
+    console.log(
+      JSON.stringify({
+        apiEndpoint: loginData.apiEndpoint!,
+        apiToken: loginData.apiToken!,
+        amount: amount,
+        peerId: peerId,
+        timeout: 60e3,
+      })
+    );
+    dispatch(
+      actionsAsync.openChannelThunk({
+        apiEndpoint: loginData.apiEndpoint!,
+        apiToken: loginData.apiToken!,
+        amount: amount,
+        peerId: peerId,
+        timeout: 60e3,
+      })
```

Comment:
My editor says: `Missing trailing comma - eslint` 

Do we want trailing commas in this scenarios 🤔